app/controllers/company_controller.rb

```diff
@@ -2,12 +2,12 @@
 
 class CompanyController < ApplicationController
   respond_to :json
-  before_action :access_lvl_helper, :ability_lvl_helper
   load_and_authorize_resource
+  before_action :company_by_id, only: %i[show update]
 
   def index
     companies = []
-    if @ability_lvl == 'system'
+    if @ability_lvl == UserRole::ABILITY_SYSTEM
```

Comment:
maybe you can add some method who will return `true` for check like this?
for example `ability_system?` and return `true` if `ability_lvl == UserRole::ABILITY_SYSTEM`
also try not often use instance variables